packages/libp2p/src/dcutr/dcutr.ts

```diff
@@ -0,0 +1,402 @@
+import { CodeError } from '@libp2p/interface/errors'
+import { logger } from '@libp2p/logger'
+import { type Multiaddr, multiaddr } from '@multiformats/multiaddr'
+import { Circuit, IP, DNS } from '@multiformats/multiaddr-matcher'
+import delay from 'delay'
+import { pbStream } from 'it-protobuf-stream'
+import isPrivate from 'private-ip'
+import { codes } from '../errors.js'
+import { HolePunch } from './pb/message.js'
+import {
+  multicodec
+} from './index.js'
```

Comment:
No reason, I think at some point there may have been multiple things imported.

If this is an issue, please [PR the linting rules](https://github.com/ipfs/eslint-config-ipfs) so this can be an automated fix.